test/unit/org/apache/cassandra/cql3/validation/operations/AggregationTest.java

```diff
@@ -1814,9 +1815,16 @@ public void testEmptyListAndNullInitcond() throws Throwable
     public void testLogbackReload() throws Throwable
     {
         // see https://issues.apache.org/jira/browse/CASSANDRA-11033
+        Logger l = LoggerFactory.getLogger(AggregationTest.class);
```

Comment:
the test didn't work previously due to `ReconfigureOnChangeTask` being used instead of filters.

I changed it to run the reconfigure task as intended. let's see what happens....